src/mina-signer/src/rosetta.ts

```diff
@@ -282,7 +282,7 @@ function rosettaTransactionToSignedCommand({
 }
 
 type UnsignedPayload = {
-  unsigned_transaction: UnsignedTransaction;
+  unsigned_transaction: string;
```

Comment:
What is the issue with `unsigned_transaction` being of type `UnsignedTransaction`? Is there no way to keep it typed here?